### PR TITLE
Fix route v8 multiple via points

### PR DESCRIPTION
**Pull request by cecileprat**
Regarding `route_v8` and the use of `via` parameter:

When adding more than 10 via points, the last ones are ignored.

It comes from the fact that in that case, some `keys_for_manipulation` are substrings of others (ex: `"via1"` and `"via10"`).
When you replace the first one in the url, it also replaces a part of the second one, and so the replace does not work as expected (you end up with a `via0=` in the final url)

I just ensured that first keys can not be substrings of last ones.

### CODE REVIEW

File: herepy/routing_api.py [hunk 1]
```diff
@@ -448,7 +448,7 @@ def route_v8(
         via_keys = []
         if via:
             for i, v in enumerate(via):
-                key = str.format("{0}{1}", "via", i)
+                key = str.format("{0}{1}_", "via", i)
```
Comment (abdullahselek) [inline herepy/routing_api.py:451]: What is the purpose of using `_` here?

Comment (cecileprat) [reply herepy/routing_api.py:451]: If you have 11 via points, you will have `via1` and `via10` in `data` and `manipulation_keys`

The first url built in `__get` will contain: `via1=...&via10=..`

Then we loop to replace those `manipulation_keys` by the one needed by here (`via`):

- `url.replace("via1", "via")` -> `via=...&via0=...` (it also replaces the first part of `via10`)
- `url.replace("via10", "via") `-> `via=...&via0=...` (it does nothing)

By creating `via1_` and `via10_` keys instead:

given the url containing `via1_=...&via10_=..`

- `url.replace("via1_", "via")` -> `via=...&via10_=...` (we replace only the first key)
- `url.replace("via10", "via")` -> `via=...&via=...` (we replace the second key)

Comment (abdullahselek) [reply herepy/routing_api.py:451]: Thanks for the clarification 👍 changes look good to me.

Comment (abdullahselek) [reply herepy/routing_api.py:451]: Would you mind bumping the version of `codecov` to make the CI happy?

**Review by abdullahselek — APPROVED** [review]

File: herepy/routing_api.py (hunk 1 above)
Comment (abdullahselek) [reply herepy/routing_api.py:451]: Thanks for bumping the codecov version, it seems like your test is failing. Let's fix it and then we can merge.

Comment (abdullahselek) [reply herepy/routing_api.py:451]: Responses requires Python 3.8 or higher as I see, can you remove the previous versions from CI config?

Comment (cecileprat) [reply herepy/routing_api.py:451]: do you want me to remove it from `ci.yml` and `travis.yml`, but keep python_requires=">=3.5" in setup?

Comment (abdullahselek) [reply herepy/routing_api.py:451]: Yes please, 3.5 has already reached end of life. It's a good time to stop supporting it. Please update the setup.py too.

**Comment by cecileprat** [comment]
It looks like the merge has failed and is not part of the last [release](https://github.com/abdullahselek/HerePy/releases/tag/3.6.1)

**Pull request merged**